### DISCUSSION TITLE
Is it possible to support PlantUML rendering?

### DISCUSSION THREAD

**Discussion by jzmini** [outline/outline#4242] [Ideas]
Hi, 

It is good to know mermaid diagrams are now supported. The purpose of this thread is to discuss whether more text-to-diagram facilities beyond mermaid can be supported on https://kroki.io/. It would be a great step forward to discuss the possibility of supporting another popular diagram PlantUML.

**Comment by jzmini** [comment]
I apologize for adding the wrong label "BUG" to this topic.

**Comment by PenChien** [comment]
Any update for this ideas?

**Reply by JasperSui** [reply]
Looks like this https://github.com/outline/outline/pull/5389 PR will help since we can embed a kroki URL and render the diagram by kroki.

**Comment by op8867555** [comment]
Hi there, I'm working on a POC (#6680), please feel free to give it a try and any feedback is welcomed :)
